mmv1/templates/terraform/extra_schema_entry/vertex_ai_featurestore_entitytype_feature.go.erb

```diff
@@ -0,0 +1,5 @@
+"region": {
+	Type:			schema.TypeString,
+	Computed: 		true,
+	Description: 	"The region of the EntityType.",
```

Comment:
This is true, but a more appropriate description might be "The region of the feature"

Reply:
Thanks! I've updated it in [4447948](https://github.com/GoogleCloudPlatform/magic-modules/pull/6992/commits/44479484f344cd10ffe0627aa115193655b2dd23).